src/frontend/default.conf

```diff
@@ -4,12 +4,16 @@ server {
 
   root /usr/share/nginx/html;
 
+  # Serve static files
   location / {
-      try_files $uri index.html $uri/ =404;
+      try_files $uri $uri/ /index.html;
   }
 
-  error_page 404 /404.html;
-  location = /404.html {
-      internal;
+  # Optionally, handle 404 errors by redirecting to index.html
+  error_page 404 =200 /index.html;
+  
+  location ~* \.(css|js|json|png|jpg|jpeg|gif|ico|svg|woff|woff2|ttf|eot)$ {
+      expires 30d;
+      add_header Cache-Control "public, max-age=2592000";
```

Comment:
This makes me thinking:

- we could explicitly say `index.html` wants no cache
- everything in `assets/*` can be cached basically forever instead of "only" 30 days, those assets have cache-busting names
- other assets (the ones from the `public/` dir) we have to watch out because their name always stay the same. It's easy to update a file and forget that it's cached for a long time.

Reply:
no cache for `index.html` done!

Reply:
Inspecting the network tab, cache headers on response seems to be properly set. However, I was wondering why the request has a no-cache header.
<img width="783" alt="Capture d’écran 2024-07-15 à 16 04 38" src="https://github.com/user-attachments/assets/2eba9f5e-9b86-4155-b219-053809872265">